src/pages/Dashboard/container.js

```diff
@@ -17,17 +17,22 @@ import {
 } from '../../../state/dashboard/actions';
 
 class DashboardPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.checkSearchState = this.checkSearchState.bind(this);
```

Comment:
To avoid binding you can use arrow functions

Reply:
Done.